crates/ide_assists/src/handlers/move_guard.rs

```diff
@@ -312,6 +340,7 @@ fn main() {
 fn main() {
     match 92 {
         x if x > 10 => false,
+        x => {}
         _ => true
```

Comment:
This looks a bit odd, feels like in this case we shouldn't need to add a `x => {}` arm here since there is already a `_` catch all afterwards. So I think we can do one extra special case here, that is if the arm we are in for this assist is followed by an arm that has a `_` pattern(without a guard) we can omit adding the `x => {}`.